system/spi/Kconfig

```diff
@@ -15,7 +15,7 @@ if SYSTEM_SPITOOL
 
 config SPITOOL_PROGNAME
 	string "Program name"
-	default "hello"
+	default "spi"
```

Comment:
need remove CONFIG_SPITOOL_PROGNAME from these defconfig:
boards/arm/s32k1xx/rddrone-uavcan144/configs/nsh/defconfig
boards/arm/s32k1xx/rddrone-uavcan144/configs/nshdebug/defconfig
boards/arm/s32k1xx/rddrone-uavcan146/configs/nsh/defconfig
boards/arm/s32k1xx/rddrone-uavcan146/configs/nshdebug/defconfig
otherwise the build will break:
https://github.com/apache/incubator-nuttx-apps/pull/345/checks?check_run_id=917971406

Reply:
uploaded apache/incubator-nuttx#1477.

Can I build check again after that is merged?

Reply:
Only commiter can restart the check, I will do it once apache/incubator-nuttx#1477 get merged.

Reply:
Thanks.